### ISSUE TITLE
Accessibility improvements: insufficient colour contrast, ARIA labels, and landmarks

### ISSUE THREAD

**Issue by clarkepaul** [silverstripe/doc.silverstripe.org#200]
Tested using Axe with Chrome only on the main landing page so a few other pages should be tested to make sure:
- Insufficient colour contrast (17 issues)
- No landmark used
- Inappropriate ARIA roles used

**Comment by unclecheese** [comment]
Have made several improvements:

* Increase font contrast in several areas
* aria-label tags
* add `role=` to several elements
* Most other violations are coming from the syntax highlighting. I'm not sure I want to go down the road of tweaking the code theme.

Home page Axe violations are down from 32 to 3, though.

**Comment by clarkepaul** [comment]
Good effort, cheers @unclecheese
